entropylab/api/tests/migrate_param_store_0_1_to_0_2.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
@urig please move this file into pipeline/api/tests

Reply:
Done. Thanks.